src/Kirel.Identity.Controllers/KirelAuthenticationController.cs

```diff
@@ -18,7 +19,7 @@ namespace Kirel.Identity.Controllers;
 public class KirelAuthenticationController<TAuthenticationService, TKey, TUser, TRegisterDto> : Controller
     where TAuthenticationService : KirelAuthenticationService<TKey, TUser, TRegisterDto> 
     where TKey : IComparable, IComparable<TKey>, IEquatable<TKey> 
-    where TUser : IdentityUser<TKey>, IKirelUser<TKey> 
+    where TUser : KirelIdentityUser<TKey>, IKirelUser<TKey> 
```

Comment:
Make KirelIdentityUser implement the IKirelUser interface itself and then remove IKirelUser from here

Reply:
Fixed